components/suggestion/command_provider.jsx

```diff
@@ -93,4 +112,39 @@ export default class CommandProvider extends Provider {
         }
         return false;
     }
+
+    handleWebapp(pretext, resultCallback) {
+        if (pretext.startsWith('/')) {
```

Comment:
@iomodo do you mind changing this to return early on a non slash command instead of having most of the function indented? Same with `handleMobile`
```suggestion
        if (!pretext.startsWith('/')) {
        	return false;
        }
```